.gitignore

```diff
@@ -1 +1,3 @@
 .envrc
+*.swp
+*.DS_Store
```

Comment:
These files should be included in the user-level ignore file.

Reply:
TIL. Thank you 😄